python/grass/temporal/aggregation.py

```diff
@@ -258,7 +259,7 @@ def aggregate_by_topology(
 
     msgr = get_tgis_message_interface()
 
-    dbif, connection_state_changed = init_dbif(dbif)
+    dbif, connected = init_dbif(dbif)
```

Comment:
The original name seems to be according to the init_dbif docs and code. Any reason for changing it?